apps/veil/src/widgets/shielding-ticker/index.tsx

```diff
@@ -167,7 +187,14 @@ export const ShieldingTicker = () => {
             Live Shielding Activity
           </Text>
         </div>
-        <Button density='compact' iconOnly icon={X} onClick={() => setIsVisible(false)}>
+        <Button
+          density='compact'
+          iconOnly
+          icon={X}
+          onClick={() => {
+            router.push('/portfolio?showShieldingTicker=false');
```

Comment:
suggestion (non-blocking): move the link into a function within the `pages.ts` file like the `getTradePairPath` function, so the param doesn't get lost